WooCommerce/src/main/kotlin/com/woocommerce/android/ui/coupons/edit/CouponRestrictionsViewModel.kt

```diff
@@ -42,6 +44,18 @@ class CouponRestrictionsViewModel @Inject constructor(
         triggerEvent(event)
     }
 
+    fun onMinimumAmountChanged(value: BigDecimal?) {
```

Comment:
To align with the above changes:
```suggestion
    fun onMinimumAmountChanged(value: BigDecimal) {
```